### PR TITLE
Update tenant api with connectors

### PR DESCRIPTION
**Pull request by mooreds**
Added some docs around the connector policy on tenants.

### CODE REVIEW

**Review by robotdan — COMMENTED** [review]
Thanks!

File: site/docs/v1/tech/apis/_tenant-response-body-base.adoc [hunk 1]
```diff
@@ -4,6 +4,20 @@ ifeval::[{base_field_name}!=tenant]
 The list of Tenant objects.
 endif::[]
 
+[field]#{base_field_name}.connectorPolicies# [type]#[Array]# [optional]#Optional# [since]#Available since 1.18.0#::
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/apis/_tenant-response-body-base.adoc]: OK, I also removed the 'default' attributes, as again, didn't make much sense for a response, and wasn't done for any other tenant attributes.

[the remark this answers is not in the document]
Comment (robotdan) [reply site/docs/v1/tech/apis/_tenant-response-body-base.adoc]: Yes, good catch, those also only need to be on the request.

**Comment by mooreds** [comment]
Merging this today, @robotdan

**Pull request merged**